media_driver/linux/common/ddi/media_libva_common.h

```diff
@@ -298,6 +298,8 @@ typedef struct _DDI_MEDIA_SURFACE
     uint8_t                 *pSystemShadow;           // Shadow surface in system memory
 
     uint32_t                uiMapFlag;
+
+    uint32_t                uiVariantFlag;
```

Comment:
Are there any reserved purposes for this flag? If no, maybe bool is enough for it.